setup.py

```diff
@@ -37,9 +37,8 @@
     setup_requires=['pytest-runner'],
     tests_require=tests_require,
     install_requires=[
-        'openeo>=0.7.0a1.*',
+        'openeo>=0.8.1a2.*',
         'openeo_driver>=0.10.11a1.*',
-        'openeo_udf>=1.0.0rc3',
```

Comment:
we should probably not drop the dependency on `openeo_udf` yet as existing UDF usage still depends on it. A bunch of tests also depend on it